### DISCUSSION TITLE
Vertex Paint (Mask) Not Working

### DISCUSSION THREAD

**Discussion by ArvisMag** [naverz/zepeto-studio-kor#1009] [5. Q&A]
Hellow! I am following the Zepeto tutorial about "3D - Rigging & Masking" and I can't figure out why it is not working for me.

I did as the tutorial is describing in Maya and in Blender -> at the end result is the same, with no difference in Unity wen prevuing my 3D model. It doesn't hide some body parts that have vertexes painted black.

What can be, that I am doing wrong
![screen_01](https://user-images.githubusercontent.com/6276317/178699599-6f5b9100-1e6e-47d1-9ea6-f9be032fa33b.PNG)
![screen_02](https://user-images.githubusercontent.com/6276317/178699606-0bc2b4e0-61fe-46f7-88d4-f9675b98121e.PNG)
![screen_03](https://user-images.githubusercontent.com/6276317/178699613-6cf277fc-98d0-4cc9-a247-ef32033b1205.PNG)
?

**Comment by Zepeto-TechEvangelist** [comment]
Hello, This is ZEPETO World Community.
For item related issues, please use the ZEPETO Customer Center.
https://support.zepeto.me/hc/en

Thank you.

**Reply by ArvisMag** [reply]
I was writing them before and they said to me to write my problem here.
no one is giving an answer, and I am going here in circles. :/

**Reply by Zepeto-TechEvangelist** [reply]
Oh, I'm sorry to hear that. I'll asking item support team
